test/linalg/triangular.jl

```diff
@@ -278,13 +278,13 @@ for elty1 in (Float32, Float64, Complex64, Complex128, BigFloat, Int)
                 @test full(A1 + A2) == full(A1) + full(A2)
                 @test full(A1 - A2) == full(A1) - full(A2)
 
-                # Triangular-Triangular multiplication and division
-                elty1 != BigFloat && elty2 != BigFloat && @test_approx_eq full(A1*A2) full(A1)*full(A2)
+                # Triangular-Triangular multiplication and division # should transpose rather than only conjugate transpose variants be tested here?
+                elty1 != BigFloat && elty2 != BigFloat && @test_approx_eq full(A1*A2) full(A1)*full(A2) # this test passes for BigFloats and Complex{BigFloats}; why exclude those cases?
```

Comment:
Maybe the tests were added when we didn't have generic fallbacks. You can just remove the conditions.